File: backend-system1-patron-electoral/bk_sys1_padron_electoral2/bk_sys1_padron_electoral2/Models/Votante.cs
namespace bk_sys1_padron_electoral.models
{
    using Microsoft.EntityFrameworkCore;
    using System.ComponentModel.DataAnnotations.Schema;

    [Index(nameof(CI), IsUnique = true)]
    public class Votante
    {
        public Guid Id { get; set; }
        public string CI { get; set; }
        public string NombreCompleto { get; set; }
        public string Direccion { get; set; }
        public double Lat { get; set; }
        public double Lng { get; set; }
        public string FotoCIanverso { get; set; }
        public string FotoCIreverso { get; set; }
        public string FotoVotante { get; set; }

        public int RecintoId { get; set; }
    }

}

File: frontend/src/models/votante.ts
export interface Votante {
  id: string;
  ci: string;
  nombreCompleto: string;
  direccion: string;
  fotoCIanverso?: File | null;
  fotoCIreverso?: File | null;
  fotoVotante?: File | null;
  lat: number;
  lng: number;
}

export interface VotantePublico {
  ci: string;
  nombreCompleto: string;
  nombreRecinto: string;
  mesaNumero: number;
}
